packages/@webex/internal-plugin-device/test/unit/spec/device.js

```diff
@@ -210,6 +210,63 @@ describe('plugin-device', () => {
       });
     });
 
+    describe('#register()', () => {
```

Comment:
I think a test should be added such that when ```canRegister``` throws that no internal events are sent